<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kunde_menu_form</class>
 <widget class="QWidget" name="kunde_menu_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(103, 189, 255);</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>82</y>
     <width>361</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbl_name">
        <property name="font">
         <font>
          <family>Acier DisplaySolid</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Modell</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_modell">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lbl_herkunft">
        <property name="font">
         <font>
          <family>Acier DisplaySolid</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Marke</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_marke">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lbl_menge">
        <property name="font">
         <font>
          <family>Acier DisplaySolid</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Erstzulassungsjahr</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_jahr">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="lbl_haltbarkeitsdatum_2">
        <property name="font">
         <font>
          <family>Acier DisplaySolid</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Preis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_preis">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="lbl_haltbarkeitsdatum_3">
        <property name="font">
         <font>
          <family>Acier DisplaySolid</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>PS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_ps">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="lbl_haltbarkeitsdatum">
        <property name="font">
         <font>
          <family>Acier DisplaySolid</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>KM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_km">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="lbl_haltbarkeitsdatum_4">
        <property name="font">
         <font>
          <family>Acier DisplaySolid</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Treibstoff</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_treibstoff">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>561</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../auto.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>comboBox</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
